chia/consensus/blockchain.py

```diff
@@ -298,15 +300,15 @@ async def _reconsider_peak(
         peak = self.get_peak()
         if genesis:
             if peak is None:
-                block: Optional[FullBlock] = await self.block_store.get_full_block(block_record.header_hash)
```

Comment:
Block records have a header_hash field, so it's fast. No need to cache it. Only full_blocks are slow